src/main/resources/templates/db_scripts/migration_scripts/change_progress_total_data_type.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE ${myuniversity}_${mymodule}.job_executions
+SET jsonb = jsonb_set(jsonb #- '{progress, total}', '{progress, total}', to_jsonb((jsonb -> 'progress' ->> 'total')::int));
```

Comment:
I think for new job execution, progress will be null. Need to add check if progress is not null